BillTech.php

```diff
@@ -31,12 +31,15 @@
  */
 class BillTech extends LMSPlugin
 {
-	const PLUGIN_DBVERSION = 2022012000;
+	const PLUGIN_DBVERSION = 2022012000; /* Legacy for old LMS versions */
+	const PLUGIN_DB_VERSION = 2022012000;
+	const PLUGIN_SOFTWARE_VERSION = 2022012000;
```

Comment:
YAGNI. Chyba nie ma co na wyrost dodawać złożoności. Potem zapomnimy i będzie trzeba szukać, po co to było dodane.